src/lua/luaTask.c

```diff
@@ -207,18 +213,35 @@ static bool _load_script(void)
         return true;
 }
 
+static bool user_bypass_requested(void)
+{
+        pr_info("lua: Checking for Lua runtime bypass request\r\n");
+        bool bypass = false;
+        LED_disable(ERROR_LED);
+        for (size_t i = 0; i < LUA_BYPASS_FLASH_COUNT; i++) {
+            LED_toggle(ERROR_LED);
+            if (GPIO_is_button_pressed()) {
```

Comment:
How long will this stay true if a user pushes the button?  Does it hold true while button is pressed, or is a bit set until cleared?